keel-sql/src/main/resources/db/changelog/20201218-add-environment-verify-with.yml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+- changeSet:
+    id: add-environment-verify-with
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: environment
+        columns:
+        - column:
+            name: verifications
+            type: json
+    - update:
+        tableName: environment
+        columns:
+        - column:
+            name: verifications
+            value: []
+    - addNotNullConstraint:
+        tableName: environment
+        columnName: verifications
+        columnDataType: json
```

Comment:
This caters for existing data by only adding the not-null constraint after updating existing rows. You can't have a `defaultValue` on a JSON column, apparently.

Reply:
Why not null, though?

Reply:
It's never null in the `Environment` class. It's an empty set.